kore/test/Test/Kore/Builtin/Int.hs

```diff
@@ -240,6 +245,22 @@ test_abs = testUnary absIntSymbol abs
 test_tdiv :: TestTree
 test_tdiv = testPartialBinary tdivIntSymbol tdiv
 
+test_tdiv_evaluated_arguments :: TestTree
```

Comment:
Dhis test fails without using `removeEvaluated` in `functionEvaluator`, but it does't display the same error as in #1373 .
I hope it's ok

Reply:
Yes, that's expected. That error comes from something higher up.